src/timeAgo.js

```diff
@@ -21,21 +21,17 @@ angular.module('yaru22.angular-timeago', [
       });
     }
   };
-}]).factory('nowTime', ['$window', '$rootScope', function ($window, $rootScope) {
-  var nowTime = Date.now();
-  var updateTime = function() {
-    $window.setTimeout(function() {
-      $rootScope.$apply(function(){
-        nowTime = Date.now();
-        updateTime();
-      });
-    }, 1000);
+}]).factory('nowTime', function ($timeout) {
```

Comment:
hm.. I haven't done that either. The most straightforward way to do this is making changes back to the original for this part and make another commit. So you will have total 3 commits for this PR.